### ISSUE TITLE
Feature Request: vertexLogLikRatio like mincLogLikRatio

### ISSUE THREAD

**Issue by gdevenyi** [Mouse-Imaging-Centre/RMINC#167]
Also mincLogLikRatioParametricBootstrap

**Comment by cfhammill** [comment]
No problem

**Comment by gdevenyi** [comment]
After asking, I learned about compare_models, doesn't that supercede this?

**Comment by cfhammill** [comment]
`compare_models` doesn't have the ratios or p-values, but I suppose those could be added. I'd still need to do the parametric bootstrap anyway.

**Comment by gdevenyi** [comment]
We're wanting to work on some model selection for vertexLmer's so this is currently the only way to compare models.

I'm going to investigate of I can just wrap the call, but if you know off-hand it won't work please dissuade me.

**Comment by cfhammill** [comment]
Here's a snippet you can use to extract AIC from your `vertexLmer`'s

```R
AIC_summary <-
  function(mmod)
    c(RMINC:::fixef_summary(mmod), AIC = extractAIC(mmod)[2])

vlm <-
  vertexLmer(model, data = data
         , anat = t(anat)
         , REML = FALSE
         , summary_type = AIC_summary)
```

Then your `vertexLm` objects will have an AIC column with per vertex AICs.

**Comment by gdevenyi** [comment]
>          , anat = t(anat)

Is that actually meant for an anatLmer? vertexLmer doesn't have an anat object

**Comment by cfhammill** [comment]
Yeah, I lazily copied an `anatLmer` example and forgot to change that part.

**Comment by cfhammill** [comment]
The flexible summary functions are really powerful, I just haven't done a good job of advertising them.
